.github/workflows/release.yml

```diff
@@ -6,35 +6,92 @@ on:
     - '*'
 
 jobs:
-  release:
-    name: Python 3.x on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+  release-ubuntu:
+    name: Ubuntu release
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix:
-        os: [ubuntu-latest, macOS-latest, windows-2016]
 
     steps:
     - uses: actions/checkout@v2
 
     - uses: actions/cache@v1
-      if: startsWith(matrix.os, 'ubuntu')
       with:
         path: ~/.cache/pip
         key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
         restore-keys: |
           ${{ runner.os }}-pip-
 
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+
+    - name: Install twine
+      run: |
+        python -m pip install --upgrade pip
+        pip install twine
+
+    - name: Build manylinux Python wheels
+      uses: RalfG/python-wheels-manylinux-build@v0.2.2-manylinux2014_x86_64
+      with:
+        python-versions: 'cp36-cp36m cp37-cp37m cp38-cp38'
+        build-requirements: '-e .'  # pip args
+
+    - name: Publish manylinux Python wheels
+      env:
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        twine upload --username __token__ wheelhouse/*-manylinux*.whl
+
+
+  release-osx:
+    name: MacOS release
+    runs-on: macOS-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v2
+
     - uses: actions/cache@v1
-      if: startsWith(matrix.os, 'macOS')
       with:
         path: ~/Library/Caches/pip
         key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
         restore-keys: |
           ${{ runner.os }}-pip-
 
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+        pip install -e .
+
+    # Only publish source package on MacOS to prevent PyPI file conflicts
+    # and so we get LF line endings (useful for debian source packages)
```

Comment:
Maybe say here "Only publish source package on one OS, to prevent PyPI file conflicts. Use MacOS rather than Windows to get LF rather than CRLF line endings (useful for Debian source packages), and MacOS rather than Linux, since the latter has a different build process.